mpcforces_extractor/main.py

```diff
@@ -45,5 +45,4 @@ def main():
 
 
 if __name__ == "__main__":
-    main()
```

Comment:
main now does not do the run anymore before running the front-end.
This has an impact on the cli ?!